.editorconfig

```diff
@@ -0,0 +1,23 @@
+# top-most EditorConfig file
+root = true
+
+# Unix-style newlines with a newline ending every file
+[*]
+end_of_line = lf
+insert_final_newline = true
+charset = utf-8
+indent_style = space
+indent_size = 2
```

Comment:
:ewwww:

( it's Friday, I gotta troll :trollface:  )

Reply:
Yeah I thought about changing the indents here but figured maybe that's a good first bug for someone else at some point. Just matching with what the existing code is doing. :)